src/logger.ts

```diff
@@ -1,7 +1,12 @@
+import type {WriteStream} from 'fs';
 import assert from 'assert';
 import chalk from 'chalk';
 
-const LEVEL = Object.freeze({
+export type LogFormatter = (data: LogEvent) => string;
+export interface LogContext { requestId?: string }
+export interface LogEvent { level: string, time: string, requestId?: string, msg: string }
```

Comment:
Why are they exported?

Reply:
I was wrongly thinking they needed to be to be used externally, but I guess only the function signatures will be sufficient.